tests/functional/basic/test_copy_uppercase.py

```diff
@@ -76,9 +75,6 @@ def test_simple_copy_uppercase(project):
     results = run_dbt()
     assert len(results) == 7
 
-    table_comp = TableComparison(
-        adapter=project.adapter, unique_schema=project.test_schema, database=project.database
-    )
-    table_comp.assert_many_tables_equal(
-        ["seed", "VIEW_MODEL", "INCREMENTAL", "MATERIALIZED", "GET_AND_REF"]
+    check_relations_equal(
+        project.adapter, ["seed", "VIEW_MODEL", "INCREMENTAL", "MATERIALIZED", "GET_AND_REF"]
```

Comment:
This is definitely something we could do. I'm still not sure which is the best way. All of these adapter utilities take the adapter as the first parameter, which does make it feel like it belongs on the adapter. But then we'd be mixing test-specific code into the adapter.  We could subclass the adapter classes and pull in the test code as an additional superclass. Might be hard to make mypy happy, but there's probably a way to do it. Then it would be: `project.adapter.check_relations_equal(....).`

Reply:
I don't feel so strongly about it. It makes sense as a method having to do with the adapter. Whether it's defined on the adapter class or takes the adapter as its first argument is almost a cosmetic question, and probably not worth splitting hairs over

Reply:
I'm ambivalent on this also, but it would take more work to make these functions actual methods, so we'll leave as it is for now.